cmd/collector/app/options_test.go

```diff
@@ -39,12 +39,16 @@ func TestAllOptionSet(t *testing.T) {
 		Options.PreProcessSpans(func(spans []*model.Span) {}),
 		Options.Sanitizer(func(span *model.Span) *model.Span { return span }),
 		Options.QueueSize(10),
+		Options.DynQueueSizeWarmup(1_000),
+		Options.DynQueueSizeMemory(1_024),
```

Comment:
underscores seem overkill for these small numbers

Reply:
Changed